Cargo.lock

```diff
@@ -2526,6 +2526,16 @@ version = "0.2.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "04744f49eae99ab78e0d5c0b603ab218f515ea8cfe5a456d7629ad883a3b6e7d"
 
+[[package]]
+name = "os_pipe"
+version = "1.2.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "5ffd2b0a5634335b135d5728d84c5e0fd726954b87111f7506a61c502280d982"
+dependencies = [
+ "libc",
+ "windows-sys 0.59.0",
```

Comment:
Remark [RAW_DYLIB 1/2]: `os_pipe` depends on `windows-sys` which tries to link in `windows.0.52.0.lib` unless you pass a `--cfg windows_raw_dylib`. This is already what rustc / rustc tool cargo uses. I.e. the comment

```
// By default, windows-rs depends on a native library that doesn't get copied into the
// sysroot. Passing this cfg enables raw-dylib support instead, which makes the native
// library unnecessary. This can be removed when windows-rs enables raw-dylib
// unconditionally.
```

applies here.